Mathlib/Topology/UniformSpace/Basic.lean

```diff
@@ -1120,6 +1120,15 @@ nonrec theorem UniformContinuous.comp [UniformSpace β] [UniformSpace γ] {g : 
   hg.comp hf
 #align uniform_continuous.comp UniformContinuous.comp
 
+/--If a function `T` is uniformly continuous in a uniform space `β`,
+then its `n`-th iterate `T^[n]` is also uniformly continuous.-/
+theorem uniformContinuous_ite [UniformSpace β] (T : β → β) (n : ℕ) (h : UniformContinuous T) :
+    UniformContinuous T^[n] := by
+  induction' n with n hn
+  · exact uniformContinuous_id
+  · exact Function.iterate_succ _ _ ▸ UniformContinuous.comp hn h
+#align uniform_continuous_ite uniformContinuous_ite
```

Comment:
No #align for new statements, just remove this line.

Reply:
Done.